cypress/e2e/inspectlet.spec.cy.ts

```diff
@@ -0,0 +1,28 @@
+describe(
+  'To test Inspectlet integration',
+  () => {
+    beforeEach(() => {
+      cy.visit('/');
+    });
+
+    it('Should enable integration if key was provided', ()=>{
+      cy.window().then(win => {
+        let inspectlet_key = win.inspectlet_key;
+        cy.visit('/?inspectlet_diagnostics=true');
+        if (inspectlet_key) {
+          cy.get('.inspectlet_diagnostics').should('be.visible');
+        }
+      })
```

Comment:
I'm no sure what's happening here but this test case is definitely not setup and written according to what's required.

Please rewrite the this test case, where:
- You are making sure inspectlet_key is defined.
- You are making sure `.inspectlet_diagnostics` exists.

There should be no need for any conditional expression here.